lib/src/Uncertainty/Distribution/GeneralizedPareto.cxx

```diff
@@ -2,7 +2,7 @@
 /**
  *  @brief The GeneralizedPareto distribution
  *
- *  Copyright 2005-2021 Airbus-EDF-IMACS-ONERA-Phimeca
+ *  Copyright 2005-2020 Airbus-EDF-IMACS-ONERA-Phimeca
```

Comment:
wanna stay in 2020 ?

Reply:
Do you see any change since last month ;-)?